### PR TITLE
fix: wallet data type

### PR DESCRIPTION
**Pull request by ChenRen93**

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -10,13 +10,20 @@ const BreakLine = styled.div`
   flex-basis: 100%;
 `;
 
+interface walletInfoType {
+  walletAddress: string | undefined;
```
Comment (Nebulis) [inline src/components/Header.tsx:14]: walletAddress?: string

File: src/components/Header.tsx [hunk 2]
```diff
@@ -10,13 +10,20 @@ const BreakLine = styled.div`
...
+
 export const Header: FunctionComponent = () => {
   const [isConnected, setIsConnected] = useState(false);
+
   const [wallet, setWalletInfo] = useState({
-    walletAddress: "-",
-    walletNetwork: "-",
-    walletBalance: "-",
-  });
+    walletAddress: undefined,
```
Comment (Nebulis) [inline src/components/Header.tsx:23]: {} should work with the change above

File: src/components/Header.tsx [hunk 3]
```diff
@@ -66,8 +73,8 @@ export const Header: FunctionComponent = () => {
             <div className="w-auto mb-4 lg:mb-0">
               <p className="font-medium">Current Account</p>
               <p className="break-all">
-                {wallet.walletAddress === "-" && wallet.walletAddress}
-                {wallet.walletAddress !== "-" && (
+                {wallet.walletAddress === undefined && "-"}
```
Comment (Nebulis) [inline src/components/Header.tsx:76]: dont need to use === undefined. Can use falsy assertion

!wallet.walletAddress && "-"

File: src/components/Header.tsx (hunk 1 above)
Comment (ChenRen93) [reply src/components/Header.tsx]: done. not sure why i didn't done this

File: src/components/Header.tsx (hunk 2 above)
Comment (ChenRen93) [reply src/components/Header.tsx]: done. thank you

File: src/components/Header.tsx (hunk 3 above)
Comment (ChenRen93) [reply src/components/Header.tsx]: done.

**Review by Nebulis — APPROVED** [review]

**Pull request merged**